vtex-apps/drawer
# Adding the ability to eagerly render children to improve SEO performance

#### What problem is this solving?

This PR adds the ability to eagerly render children so clients can improve SEO performance. This may come at the cost of site speed, but that's a tradeoff our clients should be able to make.

#### How to test it?


Search for `cibo secco` on the elements tab of the developer tools **on the mobile view**. You should see 10 matches, against 4 on the production website.

[Workspace](https://icarovtex--arcaplanet.myvtex.com/)
[Previous workspace (production)](https://arcaplanet.myvtex.com/)

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/0qiwSa0SwH8DTNjQyR/giphy-downsized.gif)
